docs/swagger.yaml

```diff
@@ -1773,6 +1773,109 @@ paths:
+                  example: John
+                lastName: 
+                  type: string
+                  description: Last name of user
+                  example: Doe
+                gender:
+                  type: string
+                  description: gender of the user
+                  example: non-binary
+                address:
+                  type: string
+                  description: address of the user
+                  example: 10, Ogunlana Drive, Surulere, Lagos
+                jobRole:
+                  type: string
+                  description: job role of the user
+                  example: Software Developer
+                department:
+                  type: string
+                  description: department of the user
+                  example: Software Development
+                profilePictureUrl:
+                  type: string
+                  description: profile picture url of the user
+                  example: https://imgbb.com/
+      responses:
+        '200':
+          description: Successfully updating a user
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  status:
+                    type: string
+                    description: successful updating a user
+                    example: success
+                  data:
+                    type: object
+                    description: Data for updating a user
+                    properties:
+                      id: 
+                        type: integer
+                        description: user id
+                        example: 10
+                      firstName: 
+                        type: string
+                        description: First name of user
+                        example: John
+                      lastName: 
+                        type: string
+                        description: Last name of user
+                        example: Doe
+                      email: 
+                        type: string
+                        description: Email of user
+                        example: atilade@gmail.com
+                      gender:
+                        type: string
+                        description: gender of the user
+                        example: non-binary
+                      address:
+                        type: string
+                        description: address of the user
+                        example: 10, Ogunlana Drive, Surulere, Lagos
+                      jobRole:
+                        type: string
+                        description: job role of the user
+                        example: Software Developer
+                      department:
+                        type: string
+                        description: department of the user
+                        example: Software Development  
+                      refreshToken:
```

Comment:
here too

Reply:
Oh. My bad. Thanks for the catch @jocrah

Reply:
@itzgeebee, i'm thinking we can handle that as part of this issue: https://github.com/tbf-challenge/teamwork-be/issues/242 since it kinda falls under that. That's if you want to